crates/ln-dlc-node/src/tests/load.rs

```diff
@@ -28,7 +30,13 @@ async fn single_app_many_positions_load() {
     init_tracing();
 
     let coordinator = Coordinator::new_public_regtest();
+
+    let app_event_handler = |node, event_sender| {
+        Arc::new(AppEventHandler::new(node, event_sender)) as Arc<dyn EventHandlerTrait>
+    };
```

Comment:
🔧 Can we compose this in a  `Node::start_test_app` method? It's pretty noisy atm.

Reply:
this is the only place where it's outside, because of custom requirements.

`Node::start_test_app` has different needs (which has this inside it), and cannot be used here :/

The other way I contemplated was to have `AppEventHandler::new_arc()` method that just does this for you, but then I realised it's only needed in tests; we don't rely on dynamic dispatch at all in the production code.
side note - perhaps we could refactor it away, but I can't justify the effort in doing that right now 🙃